### PR TITLE
DRY up publisher specs

### PR DESCRIPTION
**Pull request by roback**
Moved all duplicated specs from the two publishers into shared examples.

Related to the changes made in #73

In preparation of #71 (so we only have to update the specs in one place when solving it)

### CODE REVIEW

File: spec/integration/twingly/amqp/default_exchange_publisher_spec.rb [hunk 1]
```diff
@@ -1,9 +1,9 @@
 require "amqp_queue_context"
+require "publisher_examples"
 
 describe Twingly::AMQP::DefaultExchangePublisher do
   include_context "amqp queue"
-
```
Comment (walro) [inline spec/integration/twingly/amqp/default_exchange_publisher_spec.rb]: perhaps keep the newline here so it's consistent with the specs for `topic_exchange_publisher`

**Comment by walro** [comment]
Hm, Travis did not run the tests, I ran clicked on "Activate project". I wonder if more of our open source projects are like this... 🤔

**Comment by roback** [comment]
> Hm, Travis did not run the tests, I clicked on "Activate project". I wonder if more of our open source projects are like this... 🤔

I suspect it's the same for all OS projects. According to https://docs.travis-ci.com/user/migrate/open-source-repository-migration migrating repos is in beta. Don't know if "Activate project" counts as a migration though.

File: spec/integration/twingly/amqp/default_exchange_publisher_spec.rb (hunk 1 above)
Comment (roback) [reply spec/integration/twingly/amqp/default_exchange_publisher_spec.rb]: Yes, fixed in 30159a7

**Pull request merged**